packages/synapse-react-client/src/components/DataGrid/StartGridSession.tsx

```diff
@@ -171,19 +170,20 @@ export const StartGridSession = forwardRef<
     }
 
     return (
-      <>
-        <div style={{ position: 'relative', width: '300px' }}>
-          <input
-            ref={inputRef}
+      <Grid container spacing={2} sx={{ mb: 2 }}>
+        <Grid size={{ xs: 12, lg: 6 }} sx={{ position: 'relative' }}>
+          <TextField
```

Comment:
Did a quick pass at adding/swapping to MUI components just to make it easier to see everything in the layout